blueprints/modular/intro.yaml

```diff
@@ -0,0 +1,36 @@
+title: Intro
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        buttons:
+          type: tab
+          title: Intro
+          fields:
+            header.use_footer_links:
+              type: toggle
+              label: Use Footer Links
+              highlight: 1
+              help: This will let the section use the links defined in the footer section
+              options:
+                1: Yes
+                0: No
+              validate:
+                type: bool
+            header.image:
```

Comment:
Let's just let them use the Page Media section of the Content tab and have this be a description of that. For example,
```yaml
image: 
  type: display
  size: large
  label: Choosing a Custom Image
  content: Upload an image to the Page Media section of the Content tab to set the Intro image.
```

Reply:
Yeah it is definitely more consistent with how I implemented the filepicker in other blueprints. Will change.